app/assets/javascripts/admin/views/editions/_form.js

```diff
@@ -23,10 +23,18 @@
       var $radio_buttons             = $('input[type=radio]', $fieldset);
       var $minor_change_radio_button = $('#edition_minor_change_true', $fieldset);
       var $change_notes_section      = $('.js-change-notes-section', $fieldset);
+      var $change_notes_label        = $('.js-change-notes-section label');
 
       $radio_buttons.change(showOrHideChangeNotes);
+      appendHintToFormLabel();
       showOrHideChangeNotes();
 
+      function appendHintToFormLabel() {
+        var example_text = "For example, \"Edited chapter 6 - centres in Cardiff and Aberystwyth have closed.\""
+        var info_text = "The text will be published on the page and emailed to users signed up to alerts."
+        $change_notes_label.append(`<p class="hint"; style='font-weight: normal'>${example_text}<br>${info_text}<p>`);
```

Comment:
Does this JS get compiled into ES5? I think backticks are going to trip up old browsers if not.

Reply:
https://caniuse.com/#feat=template-literals

If you want to be safe, I think:

```suggestion
        $change_notes_label.append('<p class="hint"; style="font-weight: normal">' +example_text + '<br>' + info_text + '<p>');
```

Is not much worse.

Reply:
ah, very good point, thank you. I'll make the change